app/controllers/collection_drafts_controller.rb

```diff
@@ -170,6 +169,10 @@ def collection_draft_params
 
   private
 
+  def set_resource_by_model
+    set_resource(CollectionDraft.new(user: current_user, provider_id: current_user.provider_id, draft: {}))
```

Comment:
what happens when this is a proposal or a template? shouldn't you one of the more generic methods for the model name?

Reply:
I believe this is trying to solve the same problem as set_user_and_provider.  The stated reasoning during the review was that proposals do not have providers.  Reading over the feature branch, you solved that elsewhere.

Reply:
Added comment in method: We don't want to sent provider id